Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+def fail(reason) {
+  def pr_branch = ''
+  if (env.CHANGE_BRANCH != null) {
+    pr_branch = " (${env.CHANGE_BRANCH})"
+  }
+  error("${reason}")
+}
+
+node('lisk-explorer-01'){
+  try {
+    stage ('Prepare Workspace') {
+      deleteDir()
+      checkout scm
+    }
+
+    stage ('Shellcheck') {
+      try {
+        sh '''#!/bin/bash
+        # shellcheck
+        shopt -s globstar; /opt/shellcheck/shellcheck **/*.sh
```

Comment:
Any way we can make sure `shellcheck` is in the `$PATH` so that we do not have to hard code its absolute path here?